INSTALL.rst

```diff
@@ -304,13 +304,52 @@ You should also set apache to start on boot::
 
   sudo chkconfig httpd on
 
-The networking server may be started by running::
+Running the network server:
+---------------------------
 
-  haas serve_networks &
 
-as the HaaS user. To make this happen on boot, add the following to ``/etc/rc.local``::
+A systemd script for running the network server is available in the 'scripts' directory.
+Name of the script is: haas_network.service
+
+Centos:
+------
+
+Centos uses systemd to controll all its processes. 
+
+Place the file haas_network.service under:
+``/usr/lib/systemd/system/``
+
+Ubuntu:
+-------
+LTS version of Ubuntu, Ubuntu 14.04 does not come with systemd pre-installed.
+It uses "Upstart" an equivalent of systemd to manage its daemons/processes.
+
+Systemd is available from Ubuntu 15.04 onwards and LTS version 16.04 will ship with systemd by default.
+
+If you are using Ubuntu with any version prior to 15.04, you will have to install systemd before your can use the scripts provided here.
```

Comment:
For 16.04 folks should definitely be using the systemd service files. I think it might be good to have a manual setup section for reference for folks needing to set up haas on a non-systemd system, but we never actually tested the old thing on anything but CentOS/RHEL, and we should make that clear if we're going to keep them. Ultimately though, we can't document ubuntu setup without actually *doing* it. We could also add a bit suggesting writing your own upstart script if you've got the resources (and considering sending it to us if so).